docs/cudf/source/user_guide/10min.ipynb

```diff
@@ -2,20 +2,25 @@
  "cells": [
```

Comment:
<p>is the newline before <code>for more information</code> intentional? or a render issue on my end?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/12114/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='77'/>

Reply:
![Screen Shot 2022-11-17 at 5 28 34 PM](https://user-images.githubusercontent.com/11664259/202582098-5b7bc7ad-14b8-431c-9cb4-f6709a80f110.png)

Reply:
I think this is an artifact of reviewnb rendering.

Reply:
<p>But I will check if it exists: it should not.</p>

Reply:
Artifact.